services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -260,14 +258,39 @@ testGetLegalHoldDeviceStatus = do
 
     ensureQueueEmpty
 
-testRemoveLegalHoldDevice :: TestM ()
-testRemoveLegalHoldDevice = do
-    pure ()
+testDisableLegalHoldForUser :: TestM ()
+testDisableLegalHoldForUser = do
+    (owner, tid) <- createTeam
+    member <- randomUser
 
-    -- Remove Legal hold device from user.
-    -- send event to all of user's devices
-    -- send event to all communication peers
-    -- when still pending, notify clients they no longer need approval if deleted when still pending
```

Comment:
~~where are these tested? it's ok if they are not, but then they should be changed into TODOs, not removed.~~

ah, found it in the brig tests.  this is why i wanted to have all LH tests here.  can you make one comment in both places cross-referencing the other location of LH tests?

Reply:
I understand the desire to have all tests in one place, but this test is very specific to Brig (adding/removing clients); but yes, I can cross reference somehow :)